docs/core/deploying/native-aot/index.md

```diff
@@ -107,6 +107,19 @@ The preceding configuration assigns a default of `true` to the following propert
 
 These analyzers help to ensure that a library is compatible with Native AOT.
 
+### Multi-targeting for AOT compatibility
+
+When preparing libraries for AOT compatibility, if your library targets any framework earlier than `net6.0` (such as `netstandard2.0` or `net472`), you should also multi-target to `net6.0` to ensure that apps targeting `net6.0` or above get a version of your library that supports the AOT analyzers. Use the `IsTargetFrameworkCompatible` MSBuild function to conditionally enable `IsAotCompatible` for `net6.0` and above:
```

Comment:
AOT compat analysis was introduced for net8.0, so the AOT example and text should reference net8.0 instead of net6.0.